raiden/transfer/state.py

```diff
@@ -886,6 +888,7 @@ class BalanceProofSignedState(State):
         'signature',
         'sender',
         'chain_id',
+        'canonical_identifier',
```

Comment:
This should have had a migration... but i am guessing there can be a follow-up PR to add it.

Reply:
the serialization format did not change, so I don't see why it needs a migration at this point in time.

Reply:
hmm, i guess you're right.